fatiando/mesher.py

```diff
@@ -122,6 +127,10 @@ def x(self):
     def y(self):
         return self.vertices[:, 1]
 
+    def copy(self):
+        """ Return a deep copy of the current instance."""
+        return cp.deepcopy(self)
+
```

Comment:
You can remove this one since `Polygon` will inherit it from `GeometricElement`.